### PR TITLE
deploy to CP

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+              --set image.tag="${APP_VERSION}"
+
+executors:
+  deployer:
+    docker:
+      - image: circleci/python:3
+    working_directory: ~/app
+  builder:
+    docker:
+      - image: circleci/node:10-browsers
```
Comment (andrewrlee) [inline .circleci/config.yml]: bit weird being node3 with browsers when this is a ruby app (but is only really running helm command?)

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    working_directory: ~/app
+  test:
+    docker:
+      - image: circleci/node:10-browsers
+    working_directory: ~/app
```
Comment (andrewrlee) [inline .circleci/config.yml]: guess this isn't needed?

File: .circleci/config.yml (hunk 1 above)
Comment (mattops) [reply .circleci/config.yml]: Good point - yes only running helm.  Might be a plan to actually to do some testing of the code but probably not essential for this project.

File: .circleci/config.yml (hunk 2 above)
Comment (mattops) [reply .circleci/config.yml]: yes we not using the `test` executer in this config. removed.

File: dashboards/health.erb [hunk 3]
```diff
@@ -44,15 +44,18 @@ $(function() {
       <li data-row="1" data-col="12" data-sizex="1" data-sizey="1">
         <div data-id="offender-case-notes-prod" data-view="ServerStatusSquares" data-title="Offender Case Notes - Prod"></div>
       </li>
-      <li data-row="1" data-col="13" data-sizex="1" data-sizey="1">
+      <li data-row="1" data-col="12" data-sizex="1" data-sizey="1">
```
Comment (petergphillips) [inline dashboards/health.erb]: merge gone a bit haywire - I'd leave this alone otherwise there will be two 12 items

File: dashboards/health.erb [hunk 4]
```diff
@@ -44,15 +44,18 @@ $(function() {
...
-      <li data-row="1" data-col="14" data-sizex="1" data-sizey="1">
+      <li data-row="1" data-col="13" data-sizex="1" data-sizey="1">
         <div data-id="pathfinder-prod" data-view="ServerStatusSquares" data-title="Pathfinder - Prod"></div>
       </li>
-      <li data-row="1" data-col="15" data-sizex="1" data-sizey="1">
+      <li data-row="1" data-col="14" data-sizex="1" data-sizey="1">
         <div data-id="use-of-force-prod" data-view="ServerStatusSquares" data-title="Use of Force - Prod"></div>
       </li>
+      <li data-row="1" data-col="15" data-sizex="1" data-sizey="1">
+        <div data-id="offender-events-prod" data-view="ServerStatusSquares" data-title="Offender Events - Prod"></div>
```
Comment (petergphillips) [inline dashboards/health.erb]: I moved this to position 11

**Pull request merged**